Diana_Mokhnitskaya/HW7/HW7_task3.py

```diff
@@ -0,0 +1,8 @@
+def is_prime(number: [2, 1000]):
+    if 2 <= number <= 9:
+        return True
+    else:
+        return False
+
+
+print(is_prime(767))
```

Comment:
please read what is prime number: 
https://uk.wikipedia.org/wiki/%D0%9F%D1%80%D0%BE%D1%81%D1%82%D0%B5_%D1%87%D0%B8%D1%81%D0%BB%D0%BE

Reply:
I try to fix it here https://github.com/hillel-python-qa/hillel_python_qa_homeworks/pull/149/commits/b5cc51e978973e34d649f4b63d86ed342aa1bd74

Reply:
It is not work. 11 is the prime number. Your code return False